client/src/interface/Message.ts

```diff
@@ -0,0 +1,19 @@
+//for mock data UI testing, will change back to correct type with real data
+export interface Conversation {
+  conversationId: string;
+  imageURL: string | any;
```

Comment:
Why are we using type any here?

Reply:
I using type here because I am using mock data, the image is from local file, so it should be a module type in this UI now, so I will change back to string when uploading profile image to s3

Reply:
Sounds good and thanks for the clarification